### PR TITLE
Feature/misc updates

### PR DESCRIPTION
**Pull request by st3v3nhunt**
Updating to `nhsuk-bunyan-logger` brings back the ability to have varying levels of logging based either on `LOG_LEVEL` or the environment.

I've removed this repo from Codacy and followed up by getting rid of the badges in the `README` so that should be the last of that. Removed based on it not having been part of the trifecta of Coveralls, Travis and Snyk during recent conversations.

### CODE REVIEW

**Comment by c2s-dev** [comment]
:rocket: deployed to [http://connecting-to-services-pr-279.dev.c2s.nhschoices.net](http://connecting-to-services-pr-279.dev.c2s.nhschoices.net)

**Comment by c2s-dev** [comment]
:rocket: deployed to [http://connecting-to-services-pr-279.dev.c2s.nhschoices.net](http://connecting-to-services-pr-279.dev.c2s.nhschoices.net)

**Comment by c2s-dev** [comment]
:rocket: deployed to [http://connecting-to-services-pr-279.dev.c2s.nhschoices.net](http://connecting-to-services-pr-279.dev.c2s.nhschoices.net)

**Comment by st3v3nhunt** [comment]
Fixes #220 and #276

File: test/globalHooks.js [hunk 1]
```diff
@@ -9,7 +10,7 @@ before('global setup', () => {
 
 after('Check all nocks have been called', () => {
   try {
-    console.log(nock.pendingMocks());
+    log.fatal(nock.pendingMocks());
```
Comment (st3v3nhunt) [inline test/globalHooks.js]: Logged as `fatal` as the tests run with `NODE_ENV` where only `fatal` logs are output

**Pull request merged**